tket2/src/passes/chunks.rs

```diff
@@ -442,6 +444,22 @@ impl CircuitChunks {
     pub fn is_empty(&self) -> bool {
         self.chunks.is_empty()
     }
+
+    fn par_iter_mut_helper(chunk: &mut Chunk) -> &mut Circuit {
+        &mut chunk.circ
+    }
+    /// Supports implementation of rayon::iter::IntoParallelRefMutIterator
+    fn par_iter_mut(
+        &mut self,
+    ) -> rayon::iter::Map<
+        rayon::slice::IterMut<'_, Chunk>,
+        for<'a> fn(&'a mut Chunk) -> &'a mut Circuit,
+    > {
+        self.chunks
+            .as_parallel_slice_mut()
+            .into_par_iter()
+            .map(Self::par_iter_mut_helper)
+    }
```

Comment:
The helper method is more readable if inlined
```suggestion
    /// Supports implementation of rayon::iter::IntoParallelRefMutIterator
    fn par_iter_mut(
        &mut self,
    ) -> rayon::iter::Map<
        rayon::slice::IterMut<'_, Chunk>,
        for<'a> fn(&'a mut Chunk) -> &'a mut Circuit,
    > {
        self.chunks
            .as_parallel_slice_mut()
            .into_par_iter()
            .map(|chunk| &mut chunk.circ)
    }
```